src/plone/restapi/tests/test_serializer_summary.py

```diff
@@ -142,7 +142,7 @@ def test_brain_summary_includes_all_metadata_fields(self):
             summary["mime_type"] = u"text/plain"
 
         self.maxDiff = None
-        self.assertDictEqual(
+        self.assertDictContainsSubset(
```

Comment:
Unluckily `assertDictContainsSubset` (which I consider super useful is deprecated).
There are some (IMO worse) alternatives, e.g.:
https://stackoverflow.com/questions/21058230/how-to-assert-a-dict-contains-another-dict-without-assertdictcontainssubset-in-p

Reply:
Huh, I just copied it form the other place where I did a change. The others are indeed worse :(
I propose to keep it and later on we need to touch the code again to remove the deprecations anyway - in an own PR.

Reply:
Seems in Python 3 you can do:

```
# Test if d1 is a sub-dict of d2
d1.items() <= d2.items()
```

I don't like having if stmts in unit tests but that might be the best option (with six).